.storybook/data/tokens.json

```diff
@@ -88,6 +89,7 @@
   "eds-size-10": "calc(var(--eds-size-base-unit) * 10)",
   "eds-size-11": "calc(var(--eds-size-base-unit) * 11)",
   "eds-size-12": "calc(var(--eds-size-base-unit) * 12)",
+  "eds-size-20": "calc(var(--eds-size-base-unit) * 20)",
```

Comment:
Adding a value that was discussed with design. Will eventually revert these back to literal values instead of this clever factoring, as we want to allow support of tokens in other formats that aren't parsed by CSS